test/suites/container_devices_disk.sh

```diff
@@ -87,3 +89,43 @@ test_container_devices_raw_mount_options() {
   # shellcheck disable=SC2154
   deconfigure_loop_device "${loop_file_1}" "${loop_device_1}"
 }
+
+test_container_devices_disk_ceph() {
+  local LXD_BACKEND
+
+  LXD_BACKEND=$(storage_backend "$LXD_DIR")
+  if ! [ "${LXD_BACKEND}" = "ceph" ]; then
+    return
+  fi
+
+  RBD_POOL_NAME=lxdtest-$(basename "${LXD_DIR}")-disk
+  ceph osd pool create $RBD_POOL_NAME 1
+  rbd create --pool $RBD_POOL_NAME --size 50MB
+  rbd map --pool $RBD_POOL_NAME --name admin
+  RBD_POOL_PATH="/dev/rbd/${RBD_POOL_NAME}"
+  mkfs.ext4 -m0 $RBD_POOL_PATH
+  rbd unmap $RBD_POOL_PATH
+
+  lxc launch testimage ceph-disk -c security.privileged=true
+  lxc config device add ceph-disk rbd disk source=ceph:${RBD_POOL_NAME}/my-volume ceph.user_name=admin ceph.cluster_name=ceph path=/ceph
+  lxc exec ceph-disk -- stat /ceph/lost+found
+  lxc restart ceph-disk
+  lxc exec ceph-disk -- stat /cephfs
+  lxc delete -f ceph-disk
+}
+
+test_container_devices_disk_cephfs() {
+  local LXD_BACKEND
+
+  LXD_BACKEND=$(storage_backend "$LXD_DIR")
+  if ! [ "${LXD_BACKEND}" = "ceph" ]|| [ -z "${LXD_CEPH_CEPHFS:-}" ]; then
```

Comment:
We prefer to use `!=` in this case rather than the `!` in front of the block.
Also you're missing a space before the `||`.